pystxmcontrol/utils/writeNX.py

```diff
@@ -418,97 +322,40 @@ def startOutput(self):
         print("Starting output of file %s" %self.file_name)
         self.NXfile = h5py.File(self.file_name,'w',libver='latest')
         n_scan_regions = len(self.xPos)
-        if self.scan_dict["type"] == "Ptychography Image":
-            for i in range(n_scan_regions):
-                self.saveRegion(i)
+        for i in range(n_scan_regions):
+            self.saveRegion(i)
         self.NXfile.swmr_mode = True
```

Comment:
I think you have to define the SWMR at the h5py.FIie level as in: `h5py.File('name', 'r', swmr=True)`. I'm not sure what variable this is doing because it's not used again in this file.